### PR TITLE
Clean up tests

### PR DESCRIPTION
**Pull request by remk**
-     Remove unknown assets
-     Add HTTP 304 to HTTP 200 checks
-     Log HTTP errors only
-     Remove chat

Original PR from @chamerling

### CODE REVIEW

File: src/main/scala/com/linagora/openpaas/gatling/unifiedinbox/UnifiedInboxTemplateRequestsList.scala [hunk 1]
```diff
@@ -299,19 +298,19 @@ object LoginTemplateRequestsList {
...
     "/unifiedinbox/app/components/sidebar/folder-settings/folder-settings.html",
     "/unifiedinbox/views/partials/empty-messages/containers/inbox.html",
-    "/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.html",
-    "/unifiedinbox.github/app/components/sidebar/sidebar-menu-item.html",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.html",
+    //"/unifiedinbox.github/app/components/sidebar/sidebar-menu-item.html",
     "/views/modules/infinite-list/infinite-list.html",
     "/views/modules/attachment/list/item/attachment-list-item.html",
-    "/api/themes/53abdd2d5c5a7fc70a21f366/logo",
+    //"/api/themes/53abdd2d5c5a7fc70a21f366/logo",
```
Comment (rouazana) [inline src/main/scala/com/linagora/openpaas/gatling/unifiedinbox/UnifiedInboxTemplateRequestsList.scala:313]: can we get rid of the comments? we have git for that :)

or is it a reminder that we should replace this by an other asset?

File: src/main/scala/com/linagora/openpaas/gatling/core/authentication/lemonldap/LemonLdapSteps.scala [hunk 2]
```diff
@@ -51,5 +52,6 @@ object LemonLdapSteps {
   def goToOpenPaaSApplication: HttpRequestBuilder =
     http("Go to OpenPaaS application")
       .get("/")
+      //.disableFollowRedirect
```
Comment (rouazana) [inline src/main/scala/com/linagora/openpaas/gatling/core/authentication/lemonldap/LemonLdapSteps.scala:55]: remove comment?

File: src/main/scala/com/linagora/openpaas/gatling/chat/ChatTemplateRequestsList.scala [hunk 3]
```diff
@@ -169,45 +169,45 @@ object ChatTemplateRequestsList {
     "/calendar/components/fullcalendar/dist/locale-all.js",
     "/unifiedinbox/components/angularjs-dragula/dist/angularjs-dragula.js",
     "/unifiedinbox/components/sanitize-html/dist/sanitize-html.js",
-    "/unifiedinbox.twitter/components/ngtweet/dist/ngtweet.min.js",
+    //"/unifiedinbox.twitter/components/ngtweet/dist/ngtweet.min.js",
```
Comment (rouazana) [inline src/main/scala/com/linagora/openpaas/gatling/chat/ChatTemplateRequestsList.scala:172]: idem?

File: src/test/scala/com/linagora/openpaas/gatling/OpenPaaSMixSimulation.scala [hunk 4]
```diff
@@ -32,12 +32,14 @@ class OpenPaaSMixSimulation extends Simulation {
...
     .pause(1 second)
+    .exec(logout)
+    .pause(1 second)
     .during(ScenarioDuration) {
       randomSwitch(
-        25.0 -> exec(SendMessageScenari.generate()),
-        25.0 -> exec(SearchEventsScenari.generate()),
-        25.0 -> exec(SendEmailScenari.generate()),
-        25.0 -> exec(OpenContactScenari.generate())
+        //25.0 -> exec(SendMessageScenari.generate()),
```
Comment (rouazana) [inline src/test/scala/com/linagora/openpaas/gatling/OpenPaaSMixSimulation.scala:39]: same

File: src/test/resources/logback-test.xml [hunk 5]
```diff
@@ -16,7 +16,12 @@
 		</encoder>
 	</appender>
 
-	<root level="debug">
+	<logger name="io.gatling.http.engine.response" level="DEBUG">
+    <appender-ref ref="STDOUT"/>
+    <appender-ref ref="FILE"/>
```
Comment (rouazana) [inline src/test/resources/logback-test.xml:21]: indent

**Comment by rouazana** [comment]
did you check the asset list?

File: src/main/scala/com/linagora/openpaas/gatling/calendar/CalendarTemplateRequestsList.scala [hunk 6]
```diff
@@ -166,49 +166,48 @@ object CalendarTemplateRequestsList {
...
+    //"/unifiedinbox.twitter/app/components/configuration/social-networks-item.component.js",
+    //"/unifiedinbox.twitter/app/components/configuration/social-networks-item.controller.js",
+    //"/unifiedinbox.twitter/app/components/list/inbox-list-item-content/inbox-list-item-content.component.js",
+    //"/unifiedinbox.twitter/app/components/list/inbox-list-item-content/inbox-list-item-content.controller.js",
+    //"/unifiedinbox.twitter/app/components/list/twitter-list-item.component.js",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.component.js",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.component.js",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.controller.js",
+    //"/unifiedinbox.twitter/app/inbox.twitter.config.js",
+    //"/unifiedinbox.twitter/app/inbox.twitter.constants.js",
```
Comment (chibenwa) [inline src/main/scala/com/linagora/openpaas/gatling/calendar/CalendarTemplateRequestsList.scala:204]: Commented code deserves to be in the git history, not as a comment.

Can we remove it?

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by rouazana** [comment]
> did you check the asset list?

I will check that in an other ticket

**Pull request merged**